docs/setup/install/install-devtron-helm-2.md

```diff
@@ -13,6 +13,8 @@ helm install devtron devtron/devtron-operator --namespace devtroncd \
 --set secrets.POSTGRESQL_PASSWORD=change-me
 ```
 
+The above command with download `inception pod` which will trigger the installation of `Devtron` in background. {% endtab %}
```

Comment:
Move this section outside the tabbed block and combine it with installation status.